ReSharper.FSharp/src/FSharp.Common/src/Shim/TypeProviders/TypeProvidersManager.fs

```diff
@@ -52,6 +53,13 @@ type internal TypeProvidersCache() =
         if not hasValue then failwith $"Cannot get type provider {id} from TypeProvidersCache"
         else proxyTypeProvidersPerId.[id]
 
+    member x.GetProvidersLocations() =
```

Comment:
Better to use ``outputAssemblyPath`` since multiple projects can have the same ``resolutionFolder``